### PR TITLE
Update manage-settings-with-mdm-for-surface-hub.md

### PR DESCRIPTION
**Pull request by joelste**
added additional link for how to setup AAD Join with Intune

### CODE REVIEW

File: devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ You can enroll your Surface Hubs using bulk, manual, or automatic enrollment.
 
 Surface Hub now supports the ability to automatically enroll in Intune by joining the device to Azure Active Directory. 
 
-For more information, see [Enable Windows 10 automatic enrollment](https://docs.microsoft.com/intune/windows-enroll#enable-windows-10-automatic-enrollment).
+First step is to setup Automatic MDM enrollment, see [Enable Windows 10 automatic enrollment](https://docs.microsoft.com/intune/windows-enroll#enable-windows-10-automatic-enrollment).
```
Comment (illfated) [inline devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md]: (minor grammar improvement)
```suggestion
First step is to set up Automatic MDM enrollment, see [Enable Windows 10 automatic enrollment](https://docs.microsoft.com/intune/windows-enroll#enable-windows-10-automatic-enrollment).
```

File: devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md [hunk 2]
```diff
@@ -46,7 +46,9 @@ You can enroll your Surface Hubs using bulk, manual, or automatic enrollment.
...
+
+Then, when devices are setup during First-run, pick the option to join to Azure Active Directory, see [Set up admins for this device page](https://docs.microsoft.com/surface-hub/first-run-program-surface-hub#set-up-admins-for-this-device-page)
```
Comment (illfated) [inline devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md]:
```suggestion
Then, when devices are set up during First-run, pick the option to join Azure Active Directory. See [Set up admins for this device page](https://docs.microsoft.com/surface-hub/first-run-program-surface-hub#set-up-admins-for-this-device-page) for more details.
```

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md (hunk 1 above)
Comment (JohanFreelancer9) [inline devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md]:
```suggestion
First step is to set up Automatic MDM enrollment. See [Enable Windows 10 automatic enrollment](https://docs.microsoft.com/intune/windows-enroll#enable-windows-10-automatic-enrollment) for more details.
```

File: devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md (hunk 2 above)
Comment (Dansimp) [reply devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md]: fixed

File: devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md (hunk 1 above)
Comment (Dansimp) [reply devices/surface-hub/manage-settings-with-mdm-for-surface-hub.md]: fixed

**Pull request merged**